docker/pytmv1/Pipfile

```diff
@@ -5,8 +5,9 @@ name = "pypi"
 
 [packages]
 pytmv1 = "*"
+setuptools = "70.3.0"
```

Comment:
this can be removed. @adi88d is working on this in the base image

Reply:
thank you, will remove this.